public/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+	"short_name": "React App",
+	"name": "Create React App Sample",
```

Comment:
To be sure, Is this necessary to be our project name?

Reply:
Actually, we don't need this file.